src/main/java/com/vandendaelen/handles/functions/handles/GetCurrentDimension.java

```diff
@@ -20,6 +22,7 @@ public boolean impactMoodAndLoyalty() {
 
     @Override
     public MethodResult run(ConsoleTile tardis, IArguments args) {
-        return MethodResult.of(DimensionHelper.getDimensionId(tardis.getCurrentDimension()));
+        final RegistryKey<World> currentDimension = tardis.getCurrentDimension();
+        return MethodResult.of(DimensionHelper.getDimensionId(tardis.getCurrentDimension()), currentDimension.location().getPath());
```

Comment:
`currentDimension` can be use instead of calling the method again, no ?

Reply:
Oops! quite right, I'll fix that :)